src/main/java/frc/robot/RobotContainer.java

```diff
@@ -178,30 +191,54 @@ public SwerveSubsystem getDriveTrain() {
    */
   private void configureBindings() {
     // Swerve config
-    Supplier<DriveRequest> input =
+    final Supplier<DriveRequest> input =
         () ->
             SwerveSubsystem.joystickCondition(
                 new DriveInput(
                     OI.getAxisSupplier(OI.Driver.xTranslationAxis).get(),
                     OI.getAxisSupplier(OI.Driver.yTranslationAxis).get(),
                     OI.getAxisSupplier(OI.Driver.rotationAxis).get()),
                 OI.getButton(OI.Driver.highGear).getAsBoolean());
-
-    drivetrainCommandFactory.setDefaultCommand(
-        drivetrainCommandFactory.fieldOrientedDrive(input).withName("Get Axis Suppliers"));
-
+    final DoubleSupplier direction =
+        drivetrainCommandFactory.createRotationSource(OI.Driver.controller, drivetrain);
+
+    switch (DriverConstants.DRIVE_TYPE) {
+      case FIELD_ORIENTED:
+        drivetrainCommandFactory.setDefaultCommand(
+            drivetrainCommandFactory.fieldOrientedDrive(input).withName("Field Oriented Drive"));
+        break;
+      case POINT_DRIVE:
+        drivetrainCommandFactory.setDefaultCommand(
+            drivetrainCommandFactory
+                .pointDrive(direction, SwerveSubsystem.scrubRotation(input))
+                .withName("Point Drive"));
+        break;
+      default:
+        DriverStation.reportWarning("Unknown Drive Type Selected.", false);
+        break;
+    }
     trapElvCommandFactory.setDefaultCommand(trapElvCommandFactory.stowTrapElvCommand());
 
     shooterCommandFactory.setDefaultCommand(shooterCommandFactory.shooterIdle());
 
     triggerCommandFactory.setDefaultCommand(triggerCommandFactory.getHoldCommand());
 
+    turretCommandFactory.setDefaultCommand(turretCommandFactory.idleTurret());
+
+    final TunableNumber turretTestPitch =
+        new TunableNumber("Turret Test Pitch", 0d, (a) -> {}, turretSubsystem);
+    OI.getTrigger(OI.Driver.lockAmp)
```

Comment:
Mentioned above.  THere are commands for this in SwerveDriveCommand Factory already.  Decide on one implementation.  Also I am not sure that AMP_DIRECTION should be the same for both alliances